### PR TITLE
CORE-2290 Add new pattern for stdlib loading

### PR DESCRIPTION
**Pull request by erer1243**

### CODE REVIEW

**Review by jeapostrophe — APPROVED** [review]

**Comment by jeapostrophe** [comment]
This needs to be manually rebased because of the new version

**Comment by erer1243** [comment]
rebase had no conflicts; why couldn't github do it?

File: docs/src/changelog/index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Versions and changes-within-versions are listed in reverse-chronological order:
 
 @{rcHead("0.1.12-rc.0")}
 
++ 2022/09/15: Added `{!js} new Reach(env?)`.
```
Comment (jeapostrophe) [inline docs/src/changelog/index.md:16]: I think because this is a lie? It will actually be in rc.1

Comment (jeapostrophe) [reply docs/src/changelog/index.md:16]: (I tried to modify this in github and couldn't)

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

**Comment by jeapostrophe** [comment]
I didn't merge it because the changelog is a lie

**Pull request merged**